fetch/src/test/scala/FetchTests.scala

```diff
@@ -846,9 +846,9 @@ class FetchTests extends FetchSpec {
   }
 
   "A IO action can be combined with data fetches" in {
-    def fetch[F[_]: ConcurrentEffect]: Fetch[F, List[Int]] =
+    def fetch[F[_]: Concurrent: LiftIO]: Fetch[F, List[Int]] =
       for {
-        x        <- Fetch.liftIO(IO(3))
+        x        <- Fetch.liftIO[F, Int](IO(3))
```

Comment:
This had some issues with not inferring correctly, since the `F` can't be inferred when only appearing in the return